schema.sql

```diff
@@ -362,10 +373,19 @@ CREATE TABLE IF NOT EXISTS `towns` (
   UNIQUE KEY `name` (`name`)
 ) ENGINE=InnoDB DEFAULT CHARACTER SET=utf8;
 
-INSERT INTO `server_config` (`config`, `value`) VALUES ('db_version', '31'), ('motd_hash', ''), ('motd_num', '0'), ('players_record', '0');
+CREATE TABLE IF NOT EXISTS `vipgroup_vipentry` (
```

Comment:
`account_vipgroup_entry` is a better name, it would list lexically right next to `account_vipgroups`